move/its/sources/service.move

```diff
@@ -368,24 +406,45 @@ module its::service {
     #[test]
     fun test_deploy_remote_interchain_token() {
```

Comment:
make the deploy methods into entry functions? Otherwise we'll need tickets for those as well, and i don't see the need for other packages to call it directly atm

Reply:
since they return a hot potato object (MessageTicket) we cannot. We can however make them call `send_tciket` to allow for them to be entry, what do you think?